webpack/react_app/components/RecentJobsCard/RecentJobsCard.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable camelcase */
+
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Skeleton from 'react-loading-skeleton';
+import ElipsisWithTooltip from 'react-ellipsis-with-tooltip';
+
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  OkIcon,
+  ErrorCircleOIcon,
+} from '@patternfly/react-icons/dist/js/icons';
+import {
+  PropertiesSidePanel,
+  PropertyItem,
+} from '@patternfly/react-catalog-view-extension';
```

Comment:
This will probably need a foreman-js dependency, release and packaging, right? :thinking:

Reply:
yes :)
a PR already exists in foreman-js